services/PA-accounts-service/src/user/user.service.ts

```diff
@@ -86,7 +86,10 @@ export class UserService {
     const findOneOptions = {
       id: id,
     };
+    console.log('findOneOptions', findOneOptions)
```

Comment:
Are all uses of `console.log` necessary to commit?
Or can it be limited to only a few?